src/server/utils/stateConfig.js

```diff
@@ -76,7 +77,12 @@ if (process.env.NODE_ENV === 'development' && fs.existsSync(pathToDevEndpoints))
   // eslint-disable-next-line global-require,import/no-dynamic-require
   const devEndpoints = require(pathToDevEndpoints)();
   Object.entries(devEndpoints).forEach(([configName, { devProxyPath }]) => {
-    const value = `http://${ipAddress}:${SERVICES_PORT}/${devProxyPath}`;
+    const value = url.format({
```

Comment:
What is beneficial of making this change compared to what is already there as they result in the same thing.